documentation/GETTING_STARTED.md

```diff
@@ -0,0 +1,349 @@
+3. Save track ID, time, centroids, and other information as columns.
+4. Create columns for any additional features you want to visualize.
+
+### What does the example dataset look like?
+
+Here's a preview of the raw dataset, `data.csv`:
+
+| object_id | track_id | time | centroid_x | centroid_y | area | radius | location | segmentation_path |
+| ----------- | ---------- | ------ | ------------ | ------------ | -------- | -------- | ------------------- | --- |
+| 1 | 0 | 0 | 33 | 110 | 706.9 | 15 | middle | frame_0.tiff |
+| 2 | 1 | 0 | 67 | 100 | 804.2 | 16 | middle | frame_0.tiff |
+| 3 | 2 | 0 | 100 | 108 | 804.2 | 16 | middle | frame_0.tiff |
+| 4 | 3 | 0 | 133 | 88 | 706.9 | 15 | middle | frame_0.tiff |
+| 5 | 4 | 0 | 167 | 101 | 804.2 | 16 | middle | frame_0.tiff |
+| 6 | 0 | 1 | 33 | 121 | 530.9 | 13 | bottom | frame_1.tiff |
+| 7 | 1 | 1 | 67 | 113 | 804.2 | 16 | middle | frame_1.tiff |
+| ... | ... | ... | ... | ... | ... | ... | ... | ... |
+
+Each of the segmentation images is an OME-TIFF image containing the IDs of the segmented objects:
+
+_Frame 0 of the example dataset, as viewed in FIJI. Contrast has been increased for easier viewing._
+
+![Frame 0 of the example dataset. The background is black, labeled `value=0`. There are five circles of various diameters and positions, with IDs starting at 30 and increasing to 34.](./getting_started_guide/assets/sample-segmentation.png)
+
+> **_NOTE:_** Note that `value=0` is used to represent the background in the segmentation images. For simplicity, we recommend starting object IDs at `1` to avoid conflicts with the background value.
+
+## 4. Processing data
+
+Timelapse Feature Explorer reads data in the format specified by the [`DATA_FORMAT`](./documentation/DATA_FORMAT.md) document. We'll use the utilities provided by `colorizer-data` to convert to this format.
+
+### Processing script
+
+Start an interactive Python session. Make sure you are at `./documentation/getting_started_guide/` in the cloned repository.
+
+```bash
+# If not already in the `getting_started_guide` directory:
+cd /documentation/getting_started_guide/
+
+python
+```
+
+Paste the following steps into the terminal. (Alternatively, you can also create a Python script, copy in the code below, and run it. The full script can be found as [`process_data.py` in the `scripts` directory](./getting_started_guide/scripts/process_data.py).)
+
+#### 1. Import dependencies and load the dataset into a pandas DataFrame
+
+```python
+from bioio import BioImage
+import pandas as pd
+
+from colorizer_data.types import DATETIME_FORMAT
+from colorizer_data.utils import (
+    remap_segmented_image,
+)
+from colorizer_data.writer import (
+    ColorizerDatasetWriter,
+    ColorizerMetadata,
+    FeatureInfo,
+    FeatureType,
+)
+
+# Load the dataset
+data: pd.DataFrame = pd.read_csv("raw_dataset/data.csv")
+```
+
+#### 2. Configure the writer and data columns
+
+```python
+# Define column names
```

Comment:
```suggestion
# Define column names as they appear in the loaded csv.  Some of these are required and others are the extra calculated features for use in the colorizer.
```
I suggest adding very brief comments saying which are optional, required, and colorizable features.  Maybe good to say it elsewhere but right here is where people will be looking at the code so a good place to reinforce it.